templates/my_study/tests/conftest.py

```diff
@@ -41,7 +41,7 @@ def concept_discovery_dict(stage_type):
         f'Invalid stage type: {stage_type}. Must be one of {stage_types}')
```

Comment:
I don't like having this check here. It's just another thing to remember to update when we add output to the load stage

Reply:
oh sorry meant to get rid of that